mantidproject/mantid
# New ResNorm algorithm

Fixes #12541.

This replaces the ResNorm routine that was previously in Fortran.

To test:
- Run with following parameters
  - ResolutionWorkspace: irs26173_graphite002_res.nxs
  - VanadiumWorkspace: irs26173_graphite002_red.nxs
- See that the results are sensible
  - Stretch is a fitted stretch in the X axis between resolution and vanadium
  - Intensity is absolute intensity of resolution fitted to vanadium

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24297&oldid=24279).
